gravitee-am-certificate/gravitee-am-certificate-api/src/main/java/io/gravitee/am/certificate/api/DefaultTrustStoreProvider.java

```diff
@@ -16,24 +16,19 @@
 package io.gravitee.am.certificate.api;
 
 
-public final class DefaultTrustStoreProvider {
-    private static final String JAVA_DEFAULT_TRUSTSTORE_PROPERTY = "javax.net.ssl.trustStore";
-    private static final String JAVA_DEFAULT_TRUSTSTORE_PASSWORD_PROPERTY = "javax.net.ssl.trustStorePassword";
-    private static final String UNIFORMED_FILE_PREFIX = "file:";
-
-    private DefaultTrustStoreProvider() {
-    }
+import lombok.experimental.UtilityClass;
 
-    public static String defaultTrustStorePassword() {
-        return System.getProperty(JAVA_DEFAULT_TRUSTSTORE_PASSWORD_PROPERTY);
-    }
+@UtilityClass
+public final class DefaultTrustStoreProvider {
```

Comment:
using @UtilityClass, you dont need to set this class as final, as well as all fields as a static. https://projectlombok.org/features/experimental/UtilityClass

Reply:
agreed to leave as it is.